goose.go

```diff
@@ -2045,8 +2109,67 @@ func (ctx Ctx) imports(d []ast.Spec) []coq.Decl {
 func (ctx Ctx) maybeDecls(d ast.Decl) []coq.Decl {
 	switch d := d.(type) {
 	case *ast.FuncDecl:
+		cv := coq.StructToInterface{}
```

Comment:
I guess for now this code can only detect one struct to interface conversion in a function. That's fine for now, but it should be documented with a TODO and I think you should return an unsupported error if you encounter another required conversion.